test/integration-ebpf/src/tc_name_limit.rs

```diff
@@ -0,0 +1,28 @@
+#![no_std]
+#![no_main]
+
+use aya_bpf::{macros::classifier, programs::TcContext};
+
+// This macro generates a function with arbitrary name
+macro_rules! generate_ebpf_function {
```

Comment:
why is this macro needed? is this something forward-looking?

Reply:
But you still write it =/

Reply:
I just found it cleaner to pass 256 'a's to a macro than writing a function with that name directly. If that's an issue, I can remove it.

Reply:
Macro was replaced by just a function.